core-data/models.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { find } from 'lodash';
+
+const models = [
+	{ name: 'postType', kind: 'root', primaryKey: 'slug', baseUrl: '/wp/v2/types' },
```

Comment:
I don't know that it's an issue, but I could foresee some confusion on `primaryKey` naming with how it might be confused or misinterpreted as the database concept, where in the case of post types there is no database entity, and for others we might choose a key which isn't actually the primary key for the database entity. Maybe just me?

Reply:
Maybe just `key` ? Or `keyBy` ?